app/models/host/managed.rb

```diff
@@ -341,11 +342,14 @@ def compute?
     def compute_provides?(attr)
       false
     end
+
+    def without_orchestration
+      yield
+    end
```

Comment:
This allows calling `host.without_orchestration` even when `SETTINGS[:unattended]==false` and is the actual fix in this commit - the rest of the changes are cleanup.